### ISSUE TITLE
Make Squiz.NamingConventions.ValidVariableName to check private elements as well

### ISSUE THREAD

**Issue by TomasVotruba** [squizlabs/PHP_CodeSniffer#1851]
I'm opening this issue based on discussion on [Stackoverflow](https://stackoverflow.com/a/45539698/1348344) thanks to @gsherwood 

At the moment this sniff enforces private elements to start with `_` as well. It could be disabled by:

```xml
<rule ref="Squiz.NamingConventions.ValidVariableName.PrivateNoUnderscore">
    <severity>0</severity>
</rule>
```

But I need to check `camelCase` of private elements **as well**, not to ignore it.

As workaround [we had to create own Sniff](https://github.com/shopsys/coding-standards/pull/13/files#diff-ec379db9a2055a892de362bee3ff2e90), that does that ony for private elements.

**I'd really appreciate option that would check `camelCase` for all elements.**

/cc @vitek-rostislav

**Comment by gsherwood** [comment]
This was actually reported again as #1880 and there is a PR to fix it here: #1883

So I'll close this issue as it will be fixed by that PR.

**Comment by TomasVotruba** [comment]
Thanks for the reference and fix

**Comment by TomasVotruba** [comment]
Cc @vitek-rostislav
